packages/features/bookings/Booker/Booker.tsx

```diff
@@ -165,10 +165,6 @@ const BookerComponent = ({
 
   const shouldShowFormInDialog = shouldShowFormInDialogMap[layout];
 
-  if (bookerState === "loading") {
-    return null;
-  }
-
```

Comment:
this was blocking slots.getSchedule( inside AvailableTimeSlots) till event loading finished.

Reply:
This would cause again the flicker that was fixed [here](https://github.com/calcom/cal.com/pull/9873) 
Maybe we can keep the booker children invisible instead till the state is loading. 
https://github.com/calcom/cal.com/pull/10588/files#r1285364468

Reply:
okay thanks